src/styles/settings/_typography.scss

```diff
@@ -17,7 +17,7 @@ $weights: (
 );
 
 @mixin font( $name) {
-  @if ( map-has-key($fonts, $name) && map-has-key($weights, $name)) {
+  @if ( map-has-key($fonts, $name) & map-has-key($weights, $name)) {
```

Comment:
As in your other PR, this line should actually use `and` instead of `&`.

Reply:
Thanks! Fixed.